R/model.R

```diff
@@ -1136,19 +1145,21 @@ survey_mf <- function(survey_ids,
 #' @return Calculated prevalence
 #'
 #' @export
-anc_testing_prev_mf <- function(year, anc_testing, naomi_mf) {
+anc_testing_prev_mf <- function(year, anc_testing, naomi_mf, shape) {
```

Comment:
Refactor this so the ANC testing data aggregation only happens once, saved in something like `naomi_data$anc_aggr_input` and then attaches 4 separate columns for the inclusion / exclusion, e.g. something like: 

```
naomi_data$anc_aggr_input <- [raw_input] %>%
  mutate(
    anc_prev_t1_included = TRUE/FALSE,
    anc_prev_t2_included = TRUE/FALSE,
    anc_artcov_t1_included = TRUE/FALSE,
    anc_artcov_t2_included = TRUE/FALSE
  )
```